home-manager/common.nix

```
@@ -210,4 +210,21 @@
       "!.git"
     ];
   };
+
+  # https://github.com/nix-community/home-manager/blob/release-25.05/modules/programs/atuin.nix
+  programs.atuin = {
+    enable = true;
+
+    flags = [
+      # https://github.com/atuinsh/atuin/issues/51
+      "--disable-up-arrow"
+
+      "--disable-ctrl-r" # Keep fzf key-bindings
+    ];
+
+    settings = {
+      # auto_sync = true; # TODO: Consider enabling after test
+      sync_address = "https://algae";
```

https://github.com/kachick/dotfiles/pull/1265 might be better than tailscale

---

```suggestion
      sync_address = "https://algae.local";
```

However the mDNS is flaky